autoload/exconfig.vim

```diff
@@ -106,7 +106,7 @@ function exconfig#apply()
         let &tagrelative=0 " set notagrelative
 
         let s:old_tags=&tags
-        let &tags=fnameescape(g:exvim_folder.'/tags')
+        set tags+=fnameescape(g:exvim_folder.'/tags')
```

Comment:
the problem is, `exconfig#apply` will be invoked each time .exvim file has been saved. 
This change will make `tags` increased when the function runs.